server/src/db/index.js

```diff
@@ -1,5 +1,8 @@
 const mongoose = require('mongoose');
 
+// Removes deprecation warning when enforcing uniquness
+mongoose.set('useCreateIndex', true);
```

Comment:
I think the warning actually existed already and is not just for the uniqueness thing, I'm running current master and it shows the same deprecation warning. I'm actually not sure if it's an issue, but this does get rid of the warning. Anyone else have any ideas?

Reply:
Warning would disappear when I removed the uniqueness check. But I was working on the version before #161 was merged. Jacob has added a uniqueness check too which is why the warning will still appear

Reply:
ah right. sweet